apps/examples/testcase/ta_tc/systemio/utc/Kconfig

```diff
@@ -17,6 +17,7 @@ config SYSIO_UTC_UART
 config SYSIO_UTC_SPI
 	bool "Sysio SPI TC"
 	default n
+	select S5J_SPI
```

Comment:
When we use other board, this selection is wrong. I think "depends on" is better.